dabapush/Writer/FacebookDBWriter.py

```diff
@@ -6,11 +7,20 @@
 
 
 class FacebookDBWriter(Writer):
-    """ """
-
-    def __init__(self, config):
-
-        self.initialize_db = smo_database.DB_Manager(config)
+    """Persists Facebook data."""
+
+    def __init__(self, config: DBWriterConfiguration):
+        super().__init__(config)
+
+        self.initialize_db = smo_database.DB_Manager(
```

Comment:
@adyk007 is `smo-database`already imported/defined here?

Reply:
@adyk007 Did some quick'n'dirty tests to check whether imports and intializers are working OK. Marked as resolved.